mgo/find.go

```diff
@@ -97,12 +100,14 @@ func Find(p FindParams, results interface{}) (Cursor, error) {
 	if results == nil {
 		return Cursor{}, errors.New("results can't be nil")
 	}
+	p = ensureMandatoryParams(p)
 
-	if p.PaginatedField == "" {
-		p.PaginatedField = "_id"
-		p.Collation = nil
+	var numPaginatedFields int
+	if p.PaginatedFields != nil && len(p.PaginatedFields) > 0 {
```

Comment:
len of a non initialized slice returns 0 https://go.dev/play/p/0zdPaUDchda
```suggestion
	if len(p.PaginatedFields) > 0 {
```